techdocs/docs/using-the-package/apis-&-components/usesso.md

```diff
@@ -0,0 +1,163 @@
+# useSSO
+
+The `useSSO` hook is a custom React hook that offers authentication and authorization state and functions.
+
+As it is a React hook, you must only use it within a component or another hook.
+
+## Import
+
+```JavaScript
+// ESModule Syntax (preferred)
+import { useSSO } from "@bcgov/citz-imb-sso-react";
+
+// CommonJS Syntax
+const { useSSO } = require('@bcgov/citz-imb-sso-react');
+```
+
+## Usage
+
+### Login & Logout Buttons
+
+A basic example of using the `useSSO` hook to provide login and logout buttons.
+
+```JavaScript
+import { useSSO } from "@bcgov/citz-imb-sso-react";
+
+export const AuthButton = () => {
+  const {
+    isAuthenticated,
+    login,
+    logout,
+  } = useSSO();
+
+  return (
+    <>
+      {isAuthenticated ? (
+        <button onClick={() => logout()}>
+          LOGOUT
+        </button>
+      ) : (
+        <button onClick={() => login({ idpHint: "idir", postLoginRedirectURL: "/post-login" })}>
+          LOGIN WITH IDIR
+        </button>
+      )}
+    </>
+  );
+}
+```
+
+### Checking User's Roles
+
+A basic example of using the `useSSO` hook to display a different message based on the user's roles.
+
+```JavaScript
+import { useSSO } from "@bcgov/citz-imb-sso-react";
+
+export const RoleMessage = () => {
+  const { hasRoles } = useSSO();
+
+  // User must have 'Admin' role.
+  if (hasRoles(['Admin'])) 
+    return <p>Admin's have access to moderation features.</p>;
+
+  // Users must have BOTH 'Member' and 'Commenter' roles.
+  // requireAllRoles option is true by default.
+  if (hasRoles(['Member', 'VIP'])) 
```

Comment:
```suggestion
  if (hasRoles(['Member', 'Commenter'])) 
```

Reply:
I meant to change it to VIP but I forgot the comment. Changed that